mongomock/collection.py

```diff
@@ -1018,7 +1018,7 @@ def find(self, filter=None, projection=None, skip=0, limit=0,
              no_cursor_timeout=False, cursor_type=None, sort=None,
              allow_partial_results=False, oplog_replay=False, modifiers=None,
              batch_size=0, manipulate=True, collation=None, session=None,
-             max_time_ms=None, **kwargs):
+             max_time_ms=None, allow_disk_use=False, **kwargs):
```

Comment:
I believe the default in pymongo (at least in the doc) is None.

Reply:
Here in the doc: https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.aggregate
I can see:
`The default is False`

Reply:
My bad.